### PR TITLE
Update rancherd to point to rke2 v1.18.12+rke2r1

### PR DESCRIPTION
**Pull request by StrongMonkey**

### CODE REVIEW

**Comment by StrongMonkey** [comment]
https://github.com/rancher/rancher/issues/29825

**Review by MonzElmasry — APPROVED** [review]

File: cmd/rancherd/go.mod [hunk 1]
```diff
@@ -64,8 +64,8 @@ replace (
 
 require (
 	github.com/pkg/errors v0.9.1
-	github.com/rancher/k3s v1.19.2-0.20200930184558-c3c983198f4c
-	github.com/rancher/rke2 v1.18.11-0.20201016034716-8e418583df2c // v1.18.10+rke2r1 ... If you update this, also update rancher/package/Dockerfile.runtime
+	github.com/rancher/k3s v1.19.2-0.20201028222356-523ccaf3f22a
+	github.com/rancher/rke2 v1.18.13-0.20201116190518-b63f92a2b66e // v1.18.10+rke2r1 ... If you update this, also update rancher/package/Dockerfile.runtime
```
Comment (MonzElmasry) [inline cmd/rancherd/go.mod:68]:
```suggestion
	github.com/rancher/rke2 v1.18.13-0.20201116190518-b63f92a2b66e // v1.18.12+rke2r1 ... If you update this, also update rancher/package/Dockerfile.runtime
```

**Review by MonzElmasry — COMMENTED** [review]
need to fix the comment here

File: cmd/rancherd/go.mod (hunk 1 above)
Comment (StrongMonkey) [reply cmd/rancherd/go.mod:68]: :+1:

**Review by MonzElmasry — APPROVED** [review]

**Pull request merged**